### ISSUE TITLE
Consider MB1 GUI as option for all devices

### ISSUE THREAD

**Issue by SM0MDG** [ExpertSDR3/ExpertSDR3-SUGGESTIONS#103]
Consider including the ability to use the MB1 GUI with every EE device. This would allow for use of the small cheap touch screens available. This would be really useful for portable/mobile use and for contests and DXpeditions.

**Comment by John-KC2QMA** [comment]
It would be nice to have the MB1 GUI adapted so all devices in eSDR3 can have a "Classic Mode" to give that large scale display found in many classic transceivers of the past.

**Comment by lc7x** [comment]
I guess the suggestions above will be interesting for remote operation on e.g. a tablet.

**Comment by John-KC2QMA** [comment]
> I guess the suggestions above will be interesting for remote operation on e.g. a tablet.

Yes having a UI optimized for Touchscreen control would also be great for many devices. “Touchscreen Mode”
